middlewares/validators/user.js

```diff
@@ -173,8 +173,8 @@ async function validateUserQueryParams(req, res, next) {
       state: joi
         .alternatives()
         .try(
-          joi.string().valid("IDLE", "OOO", "ACTIVE"),
-          joi.array().items(joi.string().valid("IDLE", "OOO", "ACTIVE"))
+          joi.string().valid("IDLE", "OOO", "ACTIVE", "ONBOARDING"),
+          joi.array().items(joi.string().valid("IDLE", "OOO", "ACTIVE", "ONBOARDING"))
```

Comment:
I think it's time to move these constant strings to maybe a separate file and import the values from there.
Maybe not in this PR, but a separate one.

@prakashchoudhary07 @RitikJaiswal75 what is your opinion on this?

Reply:
+1

Reply:
Got it ! Should I resolve this for now then ?

Reply:
Please move this to a constant in this PR itself if possible.

Reply:
@naivecoder786 please create the reference issue ticket

Reply:
@RitikJaiswal75 ok will do in this PR itself

Reply:
@sahsisunny `reference issue ticket` for renaming the constants ?

Reply:
If you are doing this PR then it's ok no need to create a reference issue ticket

Reply:
> @sahsisunny `reference issue ticket` for moving the hardcoded strings to constants ?

Yes